tests/integration/test_certificates_integration.py

```diff
@@ -12,6 +12,7 @@
 
 logger = logging.getLogger(__name__)
 TEST_EXTERNAL_HOSTNAME_CONFIG = "gateway.internal"
+GATEWAY_CLASS_CONFIG = "cilium"
```

Comment:
It seems that GATEWAY_CLASS_CONFIG and TEST_EXTERNAL_HOSTNAME_CONFIG  is also declared on tests/integration/conftest.py, would it be possible to share this constant?

Reply:
yep, I imported the constant and used them instead.